content/id/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md

```diff
@@ -0,0 +1,159 @@
+{{< warning >}}Fitur-fitur Alpha berubah dengan cepat. {{< /warning >}}
+
+Network plugins in Kubernetes come in a few flavors:
+_Plugin_ jaringan di Kubernet datang dalam beberapa varian:
+
+* _Plugin_ CNI : mengikuti spesifikasi appc / CNI, yang dirancang untuk interoperabilitas.
+* _Plugin_ Kubenet : mengimplementasikan dasar `cbr0` menggunakan _plugin_ `bridge` dan `host-local` CNI
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Instalasi
+
+Kubelet memiliki _plugin_ jaringan bawaan tunggal, dan jaringan bawaan umum untuk seluruh kluster. _Plugin_ ini memeriksa _plugin-plugin_ ketika dijalankan, mengingat apa yang ditemukannya, dan mengeksekusi _plugin_ yang dipilih pada waktu yang tepat dalam siklus pod (ini hanya berlaku untuk Docker, karena rkt mengelola _plugin_ CNI sendiri). Ada dua parameter perintah Kubelet yang perlu diingat saat menggunakan _plugin_:
+
+* `cni-bin-dir`: Kubelet memeriksa direktori ini untuk _plugin-plugin_ saat startup
+* `network-plugin`: _Plugin_ jaringan untuk digunakan dari `cni-bin-dir`. Ini harus cocok dengan nama yang dilaporkan oleh _plugin_ yang diperiksa dari direktori _plugin_. Untuk _plugin_ CNI, ini hanyalah "cni".
+
+## Persyaratan _Plugin_ Jaringan
+
+Selain menyediakan [antarmuka `NetworkPlugin`](https://github.com/kubernetes/kubernetes/tree/{{< param "fullversion" >}}/pkg/kubelet/dockershim/network/plugins.go) untuk mengonfigurasi dan bersihkan jaringan Pod, plugin mungkin juga memerlukan dukungan khusus untuk kube-proxy. Proksi _iptables_ jelas tergantung pada _iptables_, dan _plugin_ mungkin perlu memastikan bahwa lalu lintas kontainer tersedia untuk _iptables_. Misalnya, jika plugin menghubungkan kontainer ke Linux _bridge_, _plugin_ harus mengatur `net/bridge/bridge-nf-call-iptables` sysctl ke` 1` untuk memastikan bahwa proksi _iptables_ berfungsi dengan benar. Jika plugin tidak menggunakan Linux _bridge_ (melainkan sesuatu seperti Open vSwitch atau mekanisme lainnya), plugin harus memastikan lalu lintas kontainer dialihkan secara tepat untuk proksi.
+
+Secara standar jika tidak ada plugin jaringan Kubelet yang ditentukan, _plugin_ `noop` digunakan, yang menetapkan `net/bridge/bridge-nf-call-iptables=1` untuk memastikan konfigurasi sederhana (seperti Docker dengan _bridge_) bekerja dengan benar dengan proksi _iptables_.
+
+### CNI
+
+_Plugin_ CNI dipilih dengan memberikan opsi _command-line_ `--network-plugin=cni` pada Kubelet. Kubelet membaca berkas dari `--cni-conf-dir` (bawaan `/etc/cni/net.d`) dan menggunakan konfigurasi CNI dari berkas tersebut untuk mengatur setiap jaringan Pod. Berkas konfigurasi CNI harus sesuai dengan [spesifikasi CNI](https://github.com/containernetworking/cni/blob/master/SPEC.md#network-configuration), dan setiap _plugin_ CNI yang diperlukan oleh konfigurasi harus ada di `--cni-bin-dir` (bawaan `/opt/cni/bin`).
+
+Jika ada beberapa berkas konfigurasi CNI dalam direktori, berkas yang digunakan adalah yang pertama dalam urutan abjad.
```

Comment:
```suggestion
Jika ada beberapa berkas konfigurasi CNI dalam direktori, Kubelet mengunakan berkas yang pertama dalam urutan abjad.
```

I found that the newest doc's already updated with this. We can update this too.
Note that this is an `alpha` page, this page might be subject to change again in the near future.

Reply:
done